libcxx/utils/libcxx/test/params.py

```diff
@@ -320,7 +320,7 @@ def getStdFlag(cfg, std):
     Parameter(
         name="executor",
         type=str,
-        default=f"{sys.executable} {Path(__file__).resolve().parent.parent.parent / 'run.py'}",
+        default=f"\"{sys.executable}\" {Path(__file__).resolve().parent.parent.parent / 'run.py'}",
```

Comment:
Could also use shutil.quote(sys.executable) to avoid unnecessary quoting?

Reply:
Yes, that’d probably work too - I’ll try that out.